src/infra/src/table/short_urls.rs

```diff
@@ -84,10 +84,7 @@ pub async fn create_table() -> Result<(), errors::Error> {
         .if_not_exists()
         .take();
 
-    client
-        .execute(builder.build(&create_table_stmt))
-        .await
-        .map_err(|e| Error::DbError(DbError::SeaORMError(e.to_string())))?;
+    client.execute(builder.build(&create_table_stmt)).await?;
```

Comment:
This logic moved to [this trait implementation](https://github.com/openobserve/openobserve/pull/5100/files#r1847373471) so it should now happen automatically when `?` converts the error.